libtenzir/include/tenzir/from_file_base.hpp

```diff
@@ -8,15 +8,14 @@
 
 #pragma once
 
-#include "tenzir/argument_parser2.hpp"
-
 #include <tenzir/actors.hpp>
+#include <tenzir/argument_parser2.hpp>
 #include <tenzir/glob.hpp>
 #include <tenzir/pipeline.hpp>
 #include <tenzir/tql2/ast.hpp>
 #include <tenzir/tql2/set.hpp>
 
-#include <arrow/filesystem/filesystem.h>
+#include <arrow/filesystem/api.h>
```

Comment:
```suggestion
#include <arrow/filesystem/type_fwd.h>
```

(the `api.h` headers are huge)